templates/checkout_cart.php

```diff
@@ -80,12 +80,12 @@
 			<tr class="edd_cart_footer_row edd_cart_subtotal_row"<?php if ( ! edd_is_cart_taxed() ) echo ' style="display:none;"'; ?>>
 				<?php do_action( 'edd_checkout_table_subtotal_first' ); ?>
 				<th colspan="<?php echo edd_checkout_cart_columns(); ?>" class="edd_cart_subtotal">
-					<?php _e( 'Subtotal', 'edd' ); ?>:&nbsp;<span class="edd_cart_subtotal_amount"><?php echo edd_cart_subtotal(); ?></span>
+					<?php echo apply_filters( 'edd_cart_table_subtotal_label', __( 'Subtotal:', 'edd' ) ); ?>&nbsp;<span class="edd_cart_subtotal_amount"><?php echo edd_cart_subtotal(); ?></span>
```

Comment:
Looks like you've moved the `:` into the translated string, this needs to be outside of it.